solutions/instances/main.tf

```diff
@@ -128,42 +128,26 @@ module "cos" {
 # SCC Instance
 #######################################################################################################################
 
-locals {
-  parsed_existing_scc_instance_crn = var.existing_scc_instance_crn != null ? split(":", var.existing_scc_instance_crn) : []
-  existing_scc_instance_guid       = length(local.parsed_existing_scc_instance_crn) > 0 ? local.parsed_existing_scc_instance_crn[7] : null
-  existing_scc_instance_region     = length(local.parsed_existing_scc_instance_crn) > 0 ? local.parsed_existing_scc_instance_crn[5] : null
-
-  scc_instance_crn    = var.existing_scc_instance_crn == null ? module.scc[0].crn : var.existing_scc_instance_crn
-  scc_instance_guid   = var.existing_scc_instance_crn == null ? module.scc[0].guid : local.existing_scc_instance_guid
-  scc_instance_region = var.existing_scc_instance_crn == null ? var.scc_region : local.existing_scc_instance_region
-
-}
-
 moved {
-  from = module.scc
-  to   = module.scc[0]
-}
-
-data "ibm_resource_instance" "scc_instance" {
-  count      = var.existing_scc_instance_crn == null ? 0 : 1
-  identifier = local.scc_instance_guid
+  from = module.scc[0]
+  to   = module.scc
 }
 
 module "scc" {
-  count                             = var.existing_scc_instance_crn == null ? 1 : 0
   source                            = "terraform-ibm-modules/scc/ibm"
-  version                           = "1.6.3"
+  version                           = "1.8.0"
+  existing_scc_instance_crn         = var.existing_scc_instance_crn
   resource_group_id                 = module.resource_group.resource_group_id
-  region                            = local.scc_instance_region
+  region                            = var.scc_region
```

Comment:
if existing instance crn is passed, then I believe the region should be parsed from CRN. why this was removed?